src/drawEngine/governors/positionGovernor/drawPositionRemovals.js

```diff
@@ -316,14 +316,14 @@ export function removeSubsequentRoundsParticipant({
       matchUp.drawPositions.includes(targetDrawPosition)
   );
 
-  relevantMatchUps?.forEach((matchUp) => {
```

Comment:
I typically like to leave the brackets because it aids in debugging when necessary, but in small/simple cases like this it is OK